dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -116,3 +116,22 @@ object reflects the newly requested size in `PersistentVolume.Spec.Capacity`. At
 this point, you can create or re-create a new pod from the PVC to finish the
 file system resizing. Once the pod is running, the newly requested size is
 available and `FileSystemResizePending` condition is removed from the PVC.
+
+[[recover_from_resize_failure]]
+== Recovering from Failure when Expanding Volumes
+
+If expanding underlying storage fails either on master or node,
+{product-title} admin can manually recover the PVC state and cancel the resize requests that are
+continuously retried by the controller without administrator intervention.
```

Comment:
Administrator intervention is needed, for the following steps need to update PV?

Reply:
I think that is clear enough.

Reply:
Ah, I misunderstand it, you want to say: if without administrator's intervention, the resize requests will continuously retried. 

Just like you said, recovering PV and PVC status is a complicated thing.
Especially for openshift 3.10, this workaround is an offline method, it maybe need the `user` and `admin/storage_admin` to do this together. For `user` knows which workloads using the PVC, so he/she is clearer about how to delete PVC, and `admin/storage_admin` has the right to update PV.

Reply:
Yeah - it is tricky thing to do and prescribing too much in these instructions can be problematic because certain steps may not apply to user's usecase and in-fact may backfire. That is why - I was thinking we should keep these docs instructional and let user/admin figure out rest of the stuff.

Reply:
Sounds good, but how about adding a note to tell user it's an offline method and if he/she does not recovery the PV and PVC's status, the Pods using the PVC still can be run continuously?